src/sentry/static/sentry/app/components/idBadge/badgeDisplayName.tsx

```diff
@@ -3,7 +3,7 @@ import styled from 'react-emotion';
 import overflowEllipsis from 'app/styles/overflowEllipsis';
 import space from 'app/styles/space';
 
-const BadgeDisplayName = styled('span')<{hideOverflow?: boolean}>`
+const BadgeDisplayName = styled('span')<{hideOverflow?: string | boolean}>`
```

Comment:
@markstory When would `hideOverflow` be a string? And what values would it be? I hope it's not like `'false'` which is truthy.